setup.py

```diff
@@ -9,9 +9,11 @@
 import setuptools
 
 
-assert os.environ.get('GITHUB_REF_TYPE') == 'tag'
-assert os.environ.get('GITHUB_REF_NAME')
-VERSION = os.environ['GITHUB_REF_NAME'].lstrip('v')
+if os.environ.get('GITHUB_REF_TYPE') == 'tag':
+    assert os.environ.get('GITHUB_REF_NAME')
+    VERSION = os.environ['GITHUB_REF_NAME'].lstrip('v')
+else:
+    VERSION = "0.0.0-dev"
```

Comment:
These changes make it possible do `pip install git+https://github.com/...`, so they do provide some value.

Reply:
@michaelkryukov thanks for following up on this!

The setup.py changes are useful to test changes without doing an release, please reconsider.

Reply:
Yeah, I've added it in separate commit already in master – https://github.com/michaelkryukov/mongomock_motor/commit/98a66c673c32e1d0a32256b124e79cd372f094e0